src/afmize.cpp

```diff
@@ -396,18 +454,42 @@ int main(int argc, char** argv)
                 outname += oss.str();
             }
 
-            afmize::write_csv (stg, outname);
-            afmize::write_json(stg, outname);
+            if(output_formats.csv)
+            {
+                afmize::write_csv (stg, outname);
+            }
+
+            if(output_formats.json)
+            {
+                afmize::write_json(stg, outname);
+            }
+
+            if(output_formats.ppm)
+            {
+            }
```

Comment:
It seems that you forgot to output ppm, no?

Reply:
I forgot to remove these lines. I ensured to output ppm with the below lines.
I'll remove these.

Reply:
👌